packages/vue-app/template/App.js

```diff
@@ -182,10 +182,25 @@ export default {
         if (page.$options.fetch && page.$options.fetch.length) {
           p.push(promisify(page.$options.fetch, this.context))
         }
-        if (page.$fetch) {
-          p.push(page.$fetch())
-        } else {
-          // Get all component instance to call $fetch
+        // get layout component
```

Comment:
Thanks for the pr!

This change seems a breaking change for layouts and fetch users. @pi0 Do you think it should be enabled with an config option flag ?

Reply:
I think this change kinda makes sense since at least current intended behavior seems refreshing every fetch and layout is not (also source of call is app). /cc @Atinux @danielroe wdyt?

I guess if it is not intended behavior, we shaol simply not do this change at all and instead document layout needs manual fetch.

Reply:
I would expect `refresh()` to refresh all fetch hooks, so I think this is a bug fix rather than a breaking change.